build.gradle

```diff
@@ -216,17 +216,6 @@ allprojects {
         implementation "com.github.spotbugs:spotbugs-annotations:${vSpotbugs}"
     }
 
-    tasks.withType(SpotBugsTask).configureEach {
-        if (name != "spotbugsMain" && System.getProperty('skipSlowTests', 'false') != 'false') {
-            enabled = false
-        }
-        // Reports can be found under each subproject in build/spotbugs/
-        reports {
-            xml.required = false
-            html.required = true
```

Comment:
can we keep the html? I found it useful when fixing those

Reply:
I've struggled to find a way to also print the console output, in addition to the HTML output. I'll try again next week.